bioblend/galaxy/invocations/__init__.py

```diff
@@ -11,9 +11,36 @@ def __init__(self, galaxy_instance):
         self.module = 'invocations'
         super().__init__(galaxy_instance)
 
-    def get_invocations(self):
+    def get_invocations(self, workflow_id=None, history_id=None, user_id=None,
+                        include_terminal=True, limit=None, view='collection',
+                        step_details=False):
         """
-        Get a list containing all workflow invocations.
+        Get a list containing all the workflow invocations corresponding to the
+        specified workflow.
```

Comment:
```suggestion
        Get all workflow invocations or those selected by filtering by some
        arguments (e.g. by workflow).
```

Not sure I like this formulation 100%, but other similar methods use something like:

```
        Get all workflow invocations or filter the specific one(s) by workflow
        or other arguments.
```

which I like even less.

Reply:
If you want a third option:
```
Get all workflow invocations, or select a subset by specifying optional arguments for filtering (e.g. a workflow ID)
```

Reply:
Third option is clearest I think.

Reply:
Thanks @simonbray , I like it! I've adopted it in lieu of the previous formulation for all `get_` methods in commit e16831f3a72b0f2c560e384693faea30fe4566a1 .